src/Microsoft.AspNet.Mvc.ModelBinding/Binders/MutableObjectModelBinder.cs

```diff
@@ -496,6 +535,82 @@ internal void ProcessDto(ModelBindingContext bindingContext, ComplexModelDto dto
             }
         }
 
+        // Neither [DefaultValue] nor [Required] is relevant for a non-settable collection.
+        private void AddToProperty(
+            ModelBindingContext bindingContext,
+            ModelExplorer modelExplorer,
```

Comment:
suggest filing a separate bug covering the use of `ModelExplorer` and both `GetExplorerForExpression()` and `GetExplorerForProperty()` in this class.  looks like this has existed pretty much forever but is more obvious since the `ModelMetadata` / `ModelExplorer` split.  note we also require a `ModelExplorer` instance to create ` `ModelValidationContext`